### DISCUSSION TITLE
[Q&A] FIPS

### DISCUSSION THREAD

**Discussion by ira-gordin-sap** [fluent/fluentd#4711] [Q&A]
### What is a problem?

Is fluentd FIPS compliant?

### Environment

```markdown
- Fluentd version:
- TD Agent version:
- Fluent Package version:
- Docker image (tag):
- Operating system:
- Kernel version:
```

**Comment by kenhys** [comment]
Since fluentd v1.18.0, `ensure_fips` option was introduced.

If your environment supports FIPS and `ensure_fips` is true, it checks the availability of FIPS compliant statue.
(It depends on OpenSSL configuration whether it is FIPS compliant)

https://www.fluentd.org/blog/fluentd-v1.18.0-has-been-released
https://docs.fluentd.org/configuration/transport-section#tls-setting
